src/main/java/graphql/schema/FieldCoordinates.java

```diff
@@ -30,6 +33,25 @@ public String getFieldName() {
         return fieldName;
     }
 
+
+    /**
+     * Checks the validity of the field coordinate names.  The validity checks vary by coordinate type.  Standard
+     * coordinates validate both the {@code typeName} and {@code fieldName}, while system coordinates do not have
+     * a parent so they only validate the {@code fieldName}.
+     *
+     * @throws AssertException if the coordinates are NOT valid; otherwise, returns normally.
+     */
+    public void assertValidNames() throws AssertException {
```

Comment:
Note:  I chose to go with a method that asserts rather than return a boolean because currently the name validation is handled by the GraphQL assert classes.  I didn't want to expose that here and I didn't want to catch the exception so I can return the boolean and then have the GraphQLCodeRegistry methods call AssertTrue(<name check>) again.